### ISSUE TITLE
[Mirrors] Modify Tencent's mirror domain

### ISSUE THREAD

**Issue by cherinyy** [msys2/msys2.github.io#297]
I noticed that MSYS2 has a built-in Tencent [mirror](https://www.msys2.org/dev/mirrors/) source domain of `mirrors.tencent.com`, which can't be said to be wrong, but it's really mostly used for web servers.

The domain used to serve the files is `mirrors.cloud.tencent.com`, which in my testing has better, global CDN support.

In [the web page](https://mirrors.tencent.com/), clicking on the `msys2` directory also redirects to https://mirrors.cloud.tencent.com/msys2/

So maybe you guys could consider modifying the domain of the Tencent mirror source.

**Comment by lazka** [comment]
Good point, https://mirrors.tencent.com/help/msys2.html also uses the other URL

**Comment by lazka** [comment]
This was part of the last mirror package update.

Thanks!
